### PR TITLE
feat(dosku-cli-github): identify contributors from GitHub

### PR DESCRIPTION
**Pull request by sidvishnoi**

### CODE REVIEW

File: services/dosku-cli-github/src/actions/identify.js [hunk 1]
```diff
@@ -0,0 +1,223 @@
...
+        const url = new URL(json.repositoryUrl);
+        repository = url.pathname.split("/").slice(0, 2).join("/");
+      } catch (error) {
+        throw new Error("Invalid repositoryUrl in context");
+      }
+    }
+  }
+  if (!repository) {
+    throw new Error("Missing repository in context");
+  }
```
Comment (sidvishnoi) [inline services/dosku-cli-github/src/actions/identify.js:34]: @mankins How do I access `dosku config` (context)? I can't find any implementation. Is it even desirable/safe, or should we pass info via CLI flags?

Comment (sidvishnoi) [reply services/dosku-cli-github/src/actions/identify.js:34]: Need to pass repo and GitHub token here

Comment (mankins) [reply services/dosku-cli-github/src/actions/identify.js:34]: Config is a "system wide" config. You can access it with `context.config` like https://github.com/LoremLabs/sendtosocial/blob/d9cc939270e98c38a073fa97ecc84a364b71b130/commands/setler-cli/src/actions/send.js#L313 ... You can access with dot paths, if I recall.

File: services/dosku-cli-github/license [hunk 2]
```diff
@@ -0,0 +1,9 @@
+MIT License
+
+Copyright (c) Matt Mankins
```
Comment (mankins) [inline services/dosku-cli-github/license:3]: Nit: should be Lorem Industries, Inc.

**Comment by mankins** [comment]
I'm going to merge this and will move to the main repo, kudos.

**Pull request merged**